app/assets/javascripts/discourse/controllers/preferences/interface.js.es6

```diff
@@ -20,6 +20,8 @@ const USER_HOMES = {
   5: "top"
 };
 
+const FONT_SIZES = ["normal", "larger", "largest"];
```

Comment:
@jjaffeux taught me a neat trick. You can add the `erb` extension on this and do `const FONT_SIZE = <%= UserOption.font_sizes.keys.map(&:to_s) %>`

Reply:
note one side effect here is that its an erb file then and caching can be a bit annoying. I am on the fence on this ...

Reply:
Why would caching be affected?

Reply:
One problem I can see is if individual sites wanted to extend this using a plugin - we can only compile one javascript file **per cluster**. 

I think if we really want to declare the sizes only on the server, we should use the Site Serializer. Writing javascript using erb templates doesn't feel like the 'right' solution to me, and might cause us problems down the line (e.g. if/when we move to ember-cli)